src/Tizen.Applications.WatchfaceComplication/Tizen.Applications.WatchfaceComplication.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.26730.16
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Tizen.Applications.WatchfaceComplication", "Tizen.Applications.WatchfaceComplication.csproj", "{F1EF8A92-02E2-483D-9D68-234640648CE3}"
```

Comment:
Maybe you should add Tizen, Tizen.Log, Tizen.Applications.Common projects to this solution file to build it also.